resources/frameworks_local.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Please remove this file.

Reply:
@limpbot those custom definitions files should ideally reside in the `user_dir`, or you can rename it locally to `my_frameworks.yaml` to have them ignored by git in this project (or create a `.gitignore` in your user home folder)